lib/builder.js

```diff
@@ -93,6 +93,35 @@ class MessageBuilder {
         return this;
     }
 
+    /**
+     * Set the embedded url
+     * @param {string} url The url 
+     */
+    setURL(url) {
+        this.data.attachments[0].url = url;
```

Comment:
Wrong key name, should be `title_link`:
```suggestion
        this.data.attachments[0].title_link = url;
```